zmon-controller-app/src/main/java/org/zalando/zmon/api/SearchAPI.java

```diff
@@ -26,11 +27,21 @@
     @Autowired
     QuickSearchSprocService searchService;
 
+    @Autowired
+    VisualizationService visualizationService;
+
+    @Autowired
+    private VisualizationProperties visualizationProperties;
```

Comment:
what's the reason behind making one field `private` and other one with default visibility?